app.js

```diff
@@ -0,0 +1,50 @@
+const bookstorage = JSON.parse(localStorage.getItem('books-display')) || [];
+const bookname = document.getElementById('book-name');
+const authorname = document.getElementById('author-name');
+const addbutton = document.querySelector('.add-book-button');
+const presentbooks = document.querySelector('.books-display');
+
+const bookobject = {
+  title: this.title,
+  author: this.author,
+  id: this.id,
+};
+
+function addbook(title, author) {
+  const newbook = Object.create(bookobject);
+  newbook.title = title;
+  newbook.author = author;
+  newbook.id = Math.floor(Math.random() * 10);
+  bookstorage.push(newbook);
+  localStorage.setItem('bookstorage', JSON.stringify(bookstorage));
+}
+
+function displaybooks() {
+  presentbooks.innerHTML = '';
+  for (let i = 0; i < bookstorage.length; i += 1) {
+    presentbooks.innerHTML += `
+      <ul>
+          <li>${bookstorage[i].title}</li>
+          <li>${bookstorage[i].author}</li>
+          <button class='re' type='button' onclick = 'removebook(${bookstorage[i].id})'>remove</button>
+      </ul>
+    `;
+  }
```

Comment:
- [ ] I have noticed that on reload, the books are not retrieved from the localStorage to display on the browser. Kindly ensure that your verify, to ensure that books are displayed on reload.